src/libslic3r/PrintConfig.cpp

```diff
@@ -525,7 +536,7 @@ void PrintConfigDef::init_common_params()
     def->label = L("Elephant foot compensation");
     def->category = L("Quality");
     def->tooltip = L("Shrinks the initial layer on build plate to compensate for elephant foot effect.");
-    def->sidetext = "mm";	// milimeters, don't need translation
+    def->sidetext = L("mm");	// unlike signs, units are translated into some languages
```

Comment:
Can we keep the PR focused on the relevant changes only? These unrelated changes make the code review extremely difficult.